src/L1VotePool.sol

```diff
@@ -7,6 +7,10 @@ abstract contract L1VotePool {
   /// @notice The address of the L1 Governor contract.
   IGovernor public governor;
 
+  event VoteCast(
+    uint256 indexed proposalId, uint256 voteAgainst, uint256 voteFor, uint256 voteAbstain
+  );
```

Comment:
It may be good to add the caller as this would more closesly match thew Governor `VoteCast` event.


https://github.com/ScopeLift/l2-flexible-voting/issues/29#issuecomment-1728168917

Reply:
done!